osf/models/metaschema.py

```diff
@@ -75,7 +75,19 @@ def validate_metadata(self, metadata, reviewer=False, required_fields=False):
         try:
             jsonschema.validate(metadata, schema)
         except jsonschema.ValidationError as e:
-            raise ValidationValueError(e.message)
+            for page in self.schema['pages']:
+                for question in page['questions']:
+                    if e.schema_path[0] == question['qid']:
+                        if 'options' in question:
+                            raise ValidationError(
+                                'For your registration the \'{}\' field is invalid, your response must be one of the provided options.'.format(
+                                    question['title'],
```

Comment:
Do you have the options here also? If so it might help to include them in the error message.

Reply:
It's hard to list every option and still be human readable. An option can be a whole sentence so just separating each list element by a comma or whatever is confusing and looks very odd to read. I don't want to use line breaks because that formatting could also look weird depending on where the message is displayed. I'd be open to other formatting ideas.

Reply:
Gotchya. My one suggestion then is to update this language just slightly - right now it says the field is invalid when in fact it's the user's response to the field that's invalid - the field being invalid is covered by the case above, correct?

Reply:
Agree with Erin, would go with "Your response to the  \'{}\' field must be one of the provided options." or similar

Reply:
Drive-by because thinking about this for file metadata: it'd be nice to have the same sorts of nicely formatted jsonschema errors wherever we raise them. I think the only difference between this and what would be good to raise for file metadata is the words "For your registration..." (which we could get rid of and it'd still make sense with maybe a bit of rewording) -- I think it might make sense to break off this jsonschema error language parsing into a function that can be reused whenever we're showing these jsonschema errors to the user. We can do this either in this PR or can be re-factored when we add the other parsing in file metadata, doesn't make a difference!